CHANGELOG.md

```diff
@@ -7,6 +7,178 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.8.0] - unreleased
 
+Fizzy passes all the official WebAssembly 1.0 tests. We are maintaining the WebAssembly 1.0 test suite
+with corrections and additions backported from the WebAssembly specification master branch. For this
+Fizzy release [the snapshot from 2022-06-23](https://github.com/wasmx/wasm-spec/tree/w3c-1.0-tests-backported-20220623/test/core) is used:
+- 19062 of 19062 binary parser and execution tests,
+- 1049 of 1049 validation tests,
+- 499 skipped due to testing text format parser.
+
+Comparing with the previous release, the performance of instantiation remains unchanged (within 1% performance regression). 
+However, we were able to increase execution performance by around **7%**.
+
+<details><summary>Detailed benchmark results</summary>
+
+```
+Comparing Fizzy 0.6.0 to 0.7.0 (Intel Haswell CPU 4.4 GHz, Clang 14)
```

Comment:
0.6.0 to 0.7.0?

@chfast This is the changelog for 0.8.0.

Reply:
Fixed.